src/main/resources/log4j2.xml

```diff
@@ -8,8 +8,22 @@
                             stackTraceEnabled="true"/>
         </Console>
         <Prometheus name="metrics"/>
+
+        <RollingFile name="securelog" filename="/Users/l153963/secure-logs/secure.log" filePattern="/Users/l153963/secure-logs/secure.log.%i" append="true">
+            <LogstashLayout dateTimeFormatPattern="yyyy-MM-dd'T'HH:mm:ss.SSSZZZ"
+                            eventTemplateUri="classpath:LogstashJsonEventLayoutV1.json"
+                            prettyPrintEnabled="false"
+                            stackTraceEnabled="true"/>
+            <ThresholdFilter level="debug" onMatch="ACCEPT" onMismatch="DENY"/>
+            <Policies>
+                <SizeBasedTriggeringPolicy size="50MB" />
```

Comment:
mm, må undersøke om det er en oppryddingspolicy så ikke poden går tom?

Reply:
Aih, såklart, dette logges jo faktisk til fil ja. 🤯

Reply:
Jeg husker vagt at Terje har lagt inn rullering av logfiler? @terjesannum

Reply:
Ser nå i https://doc.nais.io/observability/logs#log-files 

> The /secure-logs/ directory has a size limit of 128Mb, and it's the application responsibility to ensure that this limit is not exceeded. If the limit is exceeded the application pod will be evicted and restarted. Use log rotation on file size to avoid this.

Reply:
Ah .. ok

Reply:
Bruker [DefaultRolloverStrategy](https://logging.apache.org/log4j/2.x/log4j-core/apidocs/org/apache/logging/log4j/core/appender/rolling/DefaultRolloverStrategy.html) til å ta var på siste 5 logger a 25MB, den eldste blir slettet når terskelen går over